extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -101,22 +101,32 @@ function render_player( $track_list, $attributes ) {
 	ob_start();
 	?>
 	<div class="<?php echo esc_attr( $block_classname ); ?>" id="<?php echo esc_attr( $instance_id ); ?>">
-		<ol>
-			<?php
-			foreach ( $track_list as $attachment ) :
-				printf( '<li><a href="%1$s" data-jetpack-podcast-audio="%2$s">%3$s</a></li>', esc_url( $attachment['link'] ), esc_url( $attachment['src'] ), esc_html( $attachment['title'] ) );
-			endforeach;
-			?>
+		<ol class="podcast-player__episodes">
+			<?php foreach ( $track_list as $attachment ) : ?>
+			<li class="podcast-player__episode">
+				<a
+					class="podcast-player__episode-link"
+					href="<?php echo esc_url( $attachment['link'] ); ?>"
+					data-jetpack-podcast-audio="<?php echo esc_url( $attachment['src'] ); ?>"
+					role="button"
+					aria-pressed="false"
+				>
+					<span class="podcast-player__episode-status-icon"></span>
```

Comment:
It's a JS thing. This div is a placeholder for the episode state SVG icon that's rendered in.

Reply:
I updated the icon definitions to include `<title>` attributes that  essentially work as an alt text for the `svg`s. Now when you tab to the one that is playing, it will announce "Playing [track title]"